lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -81,8 +81,8 @@ def set_3d_properties(self, z=0, zdir='z'):
         self.stale = True
 
     def draw(self, renderer):
-        proj = proj3d.proj_trans_points([self._position3d, \
-                self._position3d + self._dir_vec], renderer.M)
+        proj = proj3d.proj_trans_points(
+            [self._position3d, self._position3d + self._dir_vec], renderer.M)
```

Comment:
Probably should break before the second arg though.

Reply:
the bracketed parts doesn't fit on the same line as "proj_trans_points(" and I think we have plenty of cases where the function is on one line and all the arguments on the next line.

Reply:
I mostly say to break here because the first arg is a list using commas for "other" things than separating function arguments.

Reply:
I'll follow the vote of the next reviewer.